week-8/Homework/mandatory/3-dog-photo-gallery/dog-photo.js

```diff
@@ -0,0 +1,35 @@
+let myButton = document.querySelector("#button");
+myButton.addEventListener("click",getFunctionFetch);
+function getFunctionFetch(){
+    fetch("https://dog.ceo/api/breeds/image/random")
+        .then(checkResponse)
+        .then(response => 
+            response.json()
+        )   
+        .then(data => {
+           const myImageContainer  = document.querySelector("#imageContainer")
+           let  divContainer = document.querySelector(".card-body")
+           const  myLi = document.createElement("li")
+           const myImage = document.createElement("img")
+            myImage.className += "card-img-top w-60 m-auto p-auto"
+            myImage.src = data.message
+            myLi.appendChild(myImage)
+            myImageContainer.appendChild(myLi)
+            divContainer.appendChild(myImageContainer);
+            
+        })
+        .catch((err) => {
+            console.log('Error has occurred: ' + err);
```

Comment:
A more advanced topic to think about: This is really useful for us as developers, but when the general public use websites, they tend not to look at the console, and may be confused if nothing shows up in their website. It would be interesting to think about how we could also surface errors in a way that tells the user that something has gone wrong.